tutorial_ordering.md

```diff
@@ -25,45 +25,45 @@ virtualenv <your-env>
 ```bash
 source <your-env>/bin/activate
 ```
-Now install google packages:
+Now install Google packages:
 ```bash
 pip install google
 ```
 ```bash
 pip install google-cloud-retail
 ```
 
-**Tip**: Click the copy button on the side of the code box to paste the command in the Cloud Shell terminal to run it.
+**Tip**: Click the copy button beside the code box to later paste the command in the Cloud Shell terminal and run it.
```

Comment:
if you click the icon , the command will appears promptly in the terminal, please remove "later"